test-app/app/page.tsx

```diff
@@ -0,0 +1,351 @@
+"use client";
+import React, { useState, useRef } from "react";
+import { IKImage, ImageKitProvider, IKUpload, IKVideo } from "../../src/index";
+import { AbortableFileInput, IKUploadResponse, UploadError } from "../../src/components/IKUpload/props";
+import { Transformation } from "imagekit-javascript/dist/src/interfaces/Transformation";
+
+interface ErrorType {
+  uploadFileErr: string;
+}
+
+interface OverrideParameterType {
+  fileNameOnLocalSystem: string;
+}
+
+function App() {
+  const publicKey = process.env.NEXT_PUBLIC_PUBLIC_KEY;
+  const urlEndpoint = process.env.NEXT_PUBLIC_URL_ENDPOINT;
+  const authenticationEndpoint = process.env.NEXT_PUBLIC_AUTHENTICATION_ENDPOINT;
```

Comment:
Not sure if this will work properly when doing a production build, check and verify once that once you build the bundle and change these values the sdk is picking the updated values

Reply:
We are reading values from environment variables in our example app  and not in our SDK so once the SDK is installed and the environment variable is updated user have to rebuild the app to use  updated environment variabales.